docs/guides/ingress/tcp/tcp-sni.md

```diff
@@ -46,24 +43,24 @@ spec:
         servicePort: 3443
 ```
 
-### Generated haproxy.cfg
+### Generated HAProxy.cfg
 
 ```
 # HAProxy configuration generated by https://github.com/appscode/voyager
 # DO NOT EDIT!
 global
 	daemon
-	stats socket /var/run/haproxy.sock level admin expose-fd listeners
+	stats socket /var/run/HAProxy.sock level admin expose-fd listeners
```

Comment:
copied from configmap, should not modify this...

Reply:
fixed.